packages/web-components/fast-components/src/custom-properties/behavior.ts

```diff
@@ -34,14 +34,14 @@ export function cssCustomPropertyBehaviorFactory(
         name,
         value,
         host,
-        bind(source: typeof FASTElement | HTMLElement): void {
+        bind(source: typeof FASTElement): void {
             const target = this.host(source);
 
             if (target) {
```

Comment:
Nit: Since this is perf-sensitive code, we should change `if(target)` to `if (target !== void 0`) or `null` or whatever the "empty" value is, as that explicit, non-coerced check will be slightly faster.